hikari/commands.py

```diff
@@ -136,6 +137,11 @@ class CommandOption:
     options: typing.Optional[typing.Sequence[CommandOption]] = attr.field(default=None, repr=False)
     """Sequence of up to (and including) 25 of the options for this command option."""
 
+    channel_types: typing.Optional[typing.Sequence[typing.Union[channels.ChannelType, int]]] = attr.field(
+        default=None, repr=False
+    )
+    """If a channel type option, the valid channel types that this option will accept."""
```

Comment:
`valid channel types` you shouldn't be expecting to get invalid channel types here so thats redundant

Reply:
```suggestion
    """If a channel type option, the channel types that this option will accept."""
```